tests/datasource_rubric_level.tftest.hcl

```diff
@@ -0,0 +1,98 @@
+mock_provider "opslevel" {
+  alias  = "fake"
+  source = "./mock_datasource"
+}
+
+run "datasource_rubric_level_mocked_fields" {
+  providers = {
+    opslevel = opslevel.fake
+  }
+
+  assert {
+    condition     = data.opslevel_rubric_level.id_filter.alias == "mock-rubric-level-alias"
+    error_message = "wrong alias in opslevel_rubric_level mock"
+  }
+
+  assert {
+    condition     = data.opslevel_rubric_level.id_filter.id != ""
+    error_message = "id in opslevel_rubric_level mock was not set"
+  }
+
+  assert {
+    condition     = data.opslevel_rubric_level.id_filter.index == 0
+    error_message = "index in opslevel_rubric_level should default to int type 0"
```

Comment:
What is this test useful for?

Reply:
It validates that there is an `index` in `opslevel_rubric_level` and that the `index` is a "number" type.